pos_order_to_sale_order_report/static/src/js/Popups/CreateOrderPopup.esm.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import CreateOrderPopup from "point_of_sale.CreateOrderPopup";
+import Registries from "point_of_sale.Registries";
+
+const PosDeliveryCarrierCreateOrderPopup = (CreateOrderPopup) =>
```

Comment:
Why this name ? PosDeliveryCarrierCreateOrderPopup

Reply:
ooops, sorry, the name was accidentally copied from another module, I corrected it